spec/inject_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'inject'
+
+describe Array do 
+	it 'Sum some numbers' do
+		expect([1,2,3,4,5].sebastien_injection(&:+)).to eq(15)
+	end
+	
```

Comment:
You would probably want to test a few more cases, such as multiplication, division etc.